Templates/Swift/Operation.stencil

```diff
@@ -23,11 +23,19 @@
 				case {{ variable.name|escapeReservedWord }}
 			{% endfor %}
 		}
+
 		{% if requiresCustomEncoder %}
 			{{ accessLevel }} func encode(to encoder: Encoder) throws {
+				let customScalarResolver = {{ moduleName }}.customScalarResolver
 				var container = encoder.container(keyedBy: CodingKeys.self)
 				{% for variable in operation.variables %}
-					{% with variable as field %}{% include "Helpers/EncodeField.stencil" %}{% endwith %}
```

Comment:
Why skip the `EncodeField` stencil template helper? They seem relatively similar, but the helper has more checks around nullability...

Reply:
It's my understanding that it supports null fine, but yes I suppose that would not work with undefined/optional parameters. I thought that wouldn't be a valid use case, but I suppose it's no different than being nested in an Input type. Is that correct?

Reply:
To clarify what I mean, how does one differentiate unset/optional/undefined parameters from nullable? Or is that always the case? If so, all our operations should actually be `Input<T>` , `InputWrapper<T>`.

Reply:
After getting some clarification on this from Evan, it is indeed the case that any nullable type should technically be `Input<T>` wrapped. I'm proposing that this PR is fine and unblocks @flying-turnip for the time being.

We'll have to double back later and fix our operations parameters to be optional. I'd like to explore with some alternatives before using our wrapper classes.

Reply:
Agree that we should move forward here. Not sure that I agree we should make all nullable params optional. But that's another discussion